packages/safe-apps-sdk/README.md

```diff
@@ -322,11 +322,19 @@ It is mandatory that your app exposes a `manifest.json` file in the root dir wit
 {
   "name": "YourAppName",
   "description": "A description of what your app do",
-  "iconPath": "myAppIcon.svg"
+  "icons": [
+    {
+      "src": "/myAppIcon.svg",
+      "type": "image/svg+xml",
+      "sizes": "any"
+    }
+  ]
 }
 ```
 
-> Note: iconPath it's the public relative path where the Safe will try to load your app icon. For this example, it should be https://yourAppUrl/myAppIcon.svg.
+We are following the [Manifest spec](https://developer.mozilla.org/en-US/docs/Web/Manifest) for the manifest file. `description` and `icons` are optional as is highlighted in the spec.
```

Comment:
this is not true? https://github.com/gnosis/safe-react/blob/f4714910bc7cf6aae1c844c012927bc838028389/src/routes/safe/components/Apps/__tests__/utils.test.ts#L3

Reply:
Oh, I have to fix those tests. `icons` are mandatory btw following the spec so I need to update the docs as well :(

The idea is follow the spec so make mandatory the name and icons field bu we will support as well the iconPath as fallback

Reply:
Sorry, my comment was not relevant. I had not seen other pull requests. I wanted to remove it but GitHub went down :(